pkg/service/aws/cost.go

```diff
@@ -145,6 +145,132 @@ func (svc AWSController) GetWorkspacesCost(ctx context.Context, req *proto.GetWo
 	return costResponse, nil
 }
 
+func (svc AWSController) GetApplicationsCost(ctx context.Context, req *proto.GetApplicationsCostRequest) (*proto.GetApplicationsCostResponse, error) {
+
+	session, err := NewSession(ctx, "", req.GetAccountName())
+
+	if err != nil {
+		svc.logger.Errorw("can't start AWS session", "error", err)
+		return nil, err
+	}
+
+	stsClient := session.getSTSClient()
+
+	callerIdentity, err := stsClient.GetCallerIdentity(nil)
+
+	if err != nil {
+		svc.logger.Errorw("failed to get identity", "error", err)
+		return nil, err
+	}
+
+	accound_id := callerIdentity.Account
+
+	svc.logger.Debugw("fetched accountId", "id", accound_id)
+
+	filter := &costexplorer.Expression{
```

Comment:
can we make this as a function ? will reduce the read load

Reply:
can you check now?